src/pages/ExamReviewWritePage/ExamReviewWritePage.jsx

```diff
@@ -16,7 +16,7 @@ import {
   CategoryFieldset,
   Dropdown,
 } from '@/components/Fieldset';
-import { ConfirmModal } from '@/components';
+import { ConfirmModal } from '@/components/Modal';
```

Comment:
components 안에 있는 index.js가 모든 컴포넌트들을 export 하고 있어, 기존 작성 방법이 맞는 것으로 보이는데 수정한 이유가 있을까요?? 😁 


다른 이유가 없다면, import문을 여러개로 만들지 않고 아래와 같은 방법으로 하나로 합칠 수 있을 것 같아요 :)

```suggestion
import { ActionButton, CloseAppBar, CategoryButton, CategoryFieldset, Dropdown, ConfirmModal, Icon, InputItem, InputList, Textarea } from '@/components';
```

Reply:
정확히 어느 경로에 있는 컴포넌트를 가져와 쓰는지(컴포넌트의 출처) 명확하게 알 수 없어 일부러 분리해두었습니다.